### DISCUSSION TITLE
dashboard not visible

### DISCUSSION THREAD

**Discussion by vabs212** [elsa-workflows/elsa-core#1407] [Design & tech]
![image](https://user-images.githubusercontent.com/66916559/130050065-42a44d65-80a6-440f-beeb-0c5a1471d6cb.png)
Hi i have compiled the project succesfully but on navigating to /elsa/home nothing is visible and the following screen is coming up.
Please check how elsa dashboard will be visible and how can i trigger my worflow

**Comment by vabs212** [comment]
This is my startup
![image](https://user-images.githubusercontent.com/66916559/130050599-2893a254-3622-47d9-8145-e19497e69f68.png)

**Comment by vabs212** [comment]
1) one question more do i need to add the dashboard in _host.cshtml  or this UI is fine as mentioned in link https://sipkeschoorstra.medium.com/building-workflow-driven-net-core-applications-with-elsa-139523aa4c50#:~:text=on%20the%20workflow.-,Creating%20the%20Workflow,ll%20see%20on%20the%20right.
![image](https://user-images.githubusercontent.com/66916559/130050896-74a2ceaa-8a6f-4af5-9cb6-2ab1886f1925.png)
2) .AddelsaDashboard is not present in nuget and if i install elsa.dashboard nuget for this then a runtime exception occurs.
Please suggest . i am very much confused

**Comment by vabs212** [comment]
3) Also please tell me how can i trigger my workflow for create user

**Comment by sfmskywalker** [comment]
The blog post you are referring to is about Elsa 1. But it looks like you are using Elsa 2 in your project.

I recommend you stick with Elsa 2, since Elsa 1 is no longer being developed.

[This blog post series](https://sipkeschoorstra.medium.com/building-workflow-driven-net-applications-with-elsa-2-part-1-44e08a9ba94b) was written for Elsa 2.
